hugo/layouts/report.html

```diff
@@ -294,18 +314,20 @@ <h2 class="mb-3">Your Home's Top Characteristics</h2>
 
   <!-- Comp map -->
   <h2 class="mb-3">Top 5 Most Important Sales</h2>
-  <p>Tap or click on a home to see more details.</p>
+  <p>
+    <i class="bi bi-hand-index-thumb" aria-hidden="true"></i> Tap or click on a home to see more details
```

Comment:
Now that we're using [Bootstrap icons](https://icons.getbootstrap.com/), I figure we might as well make this prompt a little bit friendlier.